feat!: add support for environment variables (#107)
open-feature/js-sdk-contrib: libs/providers/flagd/README.md

> # Server-side JavaScript flagd Provider for OpenFeature

Reviewer: #technically-correct

![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)

> Flagd is a simple command line tool for fetching and presenting feature flags to services.

Reviewer: ```suggestion
Flagd is a simple daemon for evaluating feature flags.
```

Reviewer: Maybe you don't want to accept this if it's copied from the flagd repo, but this is more accurate, I think.
